Our.Umbraco.TagHelpers/ImgTagHelper.cs

```diff
@@ -314,12 +314,63 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
             // The configuration allows us to define whether images are configured "mobile first". This simply alternates between min-width & max-width media queries.
             var imageSizes = GetImageSizes(MediaItem != null);
 
+            // Avoid rendering a WebP alternative if the image is a GIF
+            var imageFormat = MediaItem != null ? MediaItem.Url().Split('.').Last() : FileSource?.Split('.').Last(); 
+            var renderWebP = imageFormat != "gif";
+
             if (imageSizes != null && imageSizes.Any())
             {
                 var sb = new StringBuilder();
                 sb.AppendLine("<picture>");
 
                 imageSizes = _globalSettings.OurImg.MobileFirst ? imageSizes.OrderByDescending(o => o.ScreenSize).ToList() : imageSizes.OrderBy(o => o.ScreenSize).ToList();
+
+                if (renderWebP)
```

Comment:
I'd suggest moving everything in this `if` statement be moved to it's own method? Move the `renderWebP` check logic in to the method too.

```
private void RenderWebP(args) {
    if (!renderWebP) {
        return;
    }

    ...
}
```